src/main/java/mate.academy/Main.java

```diff
@@ -8,9 +8,8 @@
 public class Main {
 
     public static void main(String[] args) {
-        // Please test your Injector here. Feel free to push this class as a part of your solution
         Injector injector = Injector.getInjector();
-        ProductService productService = null;
+        ProductService productService = (ProductService) injector.getInstance(ProductService.class);
```

Comment:
Ensure that the Injector class properly handles the casting to ProductService. If the Injector cannot guarantee that the returned object is indeed an instance of ProductService, this could lead to a ClassCastException at runtime.